projects/goharbor/harbor/docker/linux/Dockerfile

```diff
@@ -0,0 +1,33 @@
+ARG BASE_IMAGE # https://gallery.ecr.aws/eks-distro-build-tooling/eks-distro-base
+FROM $BASE_IMAGE
+
+ARG RELEASE_BRANCH
+ARG TARGETARCH
+ARG TARGETOS
+
+RUN yum install -y shadow-utils >> /dev/null \
+    && yum clean all \
+    && groupadd -f -r -g 10000 chart \
+    && useradd --no-log-init -m -g 10000 -u 10000 chart
+
+COPY _output/$RELEASE_BRANCH/dependencies/$TARGETOS-$TARGETARCH/eksa/helm/chartmuseum/chartm /home/chart
+COPY ../harbor/make/photon/chartserver/docker-entrypoint.sh /home/chart/
```

Comment:
We dont do this much but i think it might be nicer if we copy these files into output_dir somewhere to avoid this.